lib/docile/fallback_context_proxy.rb

```diff
@@ -82,17 +82,25 @@ def instance_variables
 
     # Proxy all methods, excluding {NON_PROXIED_METHODS}, first to `receiver`
     # and then to `fallback` if not found.
-    def method_missing(method, *args, &block)
-      if @__receiver__.respond_to?(method.to_sym)
-        @__receiver__.__send__(method.to_sym, *args, &block)
+    args_string =
+      if RUBY_VERSION >= "2.7.0"
+        "*args, **kwargs, &block"
```

Comment:
This is not the correct way to do delegation in 2.7.
`(*args, **kwargs)`-delegation does not work in Ruby 2.7: https://eregon.me/blog/2019/11/10/the-delegation-challenge-of-ruby27.html#ruby-3-style-delegation

I'll make a PR to use `ruby2_keywords`, which is the only correct way to do delegation in Ruby 2.7.

Reply:
https://github.com/ms-ati/docile/pull/52

Reply:
Thank you @eregon !